### PR TITLE
chore(theme-translations): complete it translations

### PR DESCRIPTION
**Pull request by bitpredator**
- missing translations done

## Pre-flight checklist

- [ ] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

### CODE REVIEW

File: packages/docusaurus-theme-translations/locales/it/theme-common.json [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
+  "theme.NavBar.navAriaLabel": "Principale",
   "theme.NotFound.p1": "Non siamo riusciti a trovare quello che stavi cercando.",
   "theme.NotFound.p2": "Contatta il proprietario del sito che ti ha collegato all'URL originale e fagli sapere che il loro collegamento è interrotto.",
   "theme.NotFound.title": "Pagina non trovata",
   "theme.TOCCollapsible.toggleButtonLabel": "Su questa pagina",
-  "theme.admonition.caution": "caution",
-  "theme.admonition.danger": "danger",
-  "theme.admonition.info": "info",
+  "theme.admonition.caution": "attenzione",
+  "theme.admonition.danger": "Pericolo",
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-translations/locales/it/theme-common.json:17]:
```suggestion
  "theme.admonition.danger": "pericolo",
```

**Review by Josh-Cena — COMMENTED** [review]
Could you update the other JSON files as well? Like https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-theme-translations/locales/it/plugin-ideal-image.json

**Review by slorber — APPROVED** [review]
Thanks

Can you sign the Facebook CLA please? 
I can't merge without this step. If you already signed it the bot will eventually report it soon.

**Comment by bitpredator** [comment]
> Thanks
> 
> Can you sign the Facebook CLA please? I can't merge without this step. If you already signed it our CLA bot will eventually report it soon.

I did the Facebook CLA Check

**Pull request merged**